.github/workflows/deploy-site.yml

```
@@ -0,0 +1,44 @@
+name: Deploy site
+
+on: push
```

This won’t trigger for pull requests from non-maintainers i.e. from forked repos. 

```suggestion
on:
  pull_request:
  push:
    branches: [master]
```

---

This was kind of intentional. Firstly, it's nice to deploy the site without having to make a PR. I was on the fence about external PRs, however external PRs rarely ever need to review site changes. Could change over time though.